python/doc/examples/calibration/calibration_withoutobservedinputs.ipynb

```diff
@@ -0,0 +1,452 @@
+{
```

Comment:
Could you please convert this example to a Python script with RST commands that could work with #1574? You can use [this file](https://github.com/openturns/openturns/blob/0224e7b581878431e52e5af357def2fbdbc5d9b4/python/doc/examples/py_calibration/bayesian_calibration/plot_bayesian_calibration.py) as template.